apps/www/_blog/2022-08-03-supabase-beta-update-july-2022.mdx

```diff
@@ -57,7 +57,7 @@ We've started updating our settings UI, starting with the Auth settings. You'll
 
 ## Nuxt 3 Quickstart
 
-![nuxt-3-supabase-quickstart](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/myuix99v27hx9thob30d.png)
+![nuxt-3-supabase-quickstart](images/blog/2022-july/Nuxt Supabase.png)
```

Comment:
May want to check that it renders properly even with the space

Reply:
Yeah I noticed that, not it worked in the repo I uploaded it to.